src/Shouldly.Shared/DiffToolConfiguration.cs

```diff
@@ -0,0 +1,98 @@
+#if !PORTABLE
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Shouldly
+{
+    public class DiffToolConfiguration
+    {
+        readonly List<IShouldNotLaunchDiffTool> _knownShouldNotLaunchDiffToolReasons;
+        readonly List<DiffTool> _diffToolPriority = new List<DiffTool>();
+        readonly List<DiffTool> _diffTools;
+
+        public static class KnownDiffTools
+        {
+            public static readonly DiffTool KDiff3 = new DiffTool("KDiff3", @"C:\Program Files\KDiff3\kdiff3.exe", (received, approved) => $"\"{received}\" \"{approved}\" -o \"{approved}\"");
+        }
+
+        public static class KnownDoNoLaunchStrategies
```

Comment:
Yes, you are! Lol

Sent from Outlook Mobile




On Wed, Dec 30, 2015 at 3:07 AM -0800, "Joseph Woodward" <notifications@github.com> wrote:












In src/Shouldly.Shared/DiffToolConfiguration.cs:
> +using System.Linq;
> +
> +namespace Shouldly
> +{
> +    public class DiffToolConfiguration
> +    {
> +        readonly List<IShouldNotLaunchDiffTool> _knownShouldNotLaunchDiffToolReasons;
> +        readonly List<DiffTool> _diffToolPriority = new List<DiffTool>();
> +        readonly List<DiffTool> _diffTools;
> +
> +        public static class KnownDiffTools
> +        {
> +            public static readonly DiffTool KDiff3 = new DiffTool("KDiff3", @"C:\Program Files\KDiff3\kdiff3.exe", (received, approved) => $"\"{received}\" \"{approved}\" -o \"{approved}\"");
> +        }
> +
> +        public static class KnownDoNoLaunchStrategies



lol sorry, I wasn't trying to be picky or anything!



—
Reply to this email directly or view it on GitHub.